src/test/anovos/feature_recommender/test_featrec_init.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import pandas as pd
+from anovos.feature_recommender.featrec_init import (
+    get_column_name,
+    camel_case_split,
+    recommendation_data_prep,
+)
+
+sample_data_2 = "./data/feature_recommender/test_input_fr_2.csv"
+
+
+@pytest.fixture
+def example_attr_2():
+    return pd.read_csv(sample_data_2)
+
+
+def test_get_column_name(example_attr_2):
+    feature_name, feature_desc, industry, usecase = get_column_name(example_attr_2)
+    assert feature_name == "Name"
+    assert feature_desc == "Desc"
+    assert industry == "Industry"
+    assert usecase == "Usecase"
+
+
+def test_camel_case_split():
+    test_input_1 = "accountWeeks"
+    test_output_1 = camel_case_split(test_input_1)
+    assert test_output_1 == "account Weeks "
+
+    test_input_2 = "account Weeks"
+    test_output_2 = camel_case_split(test_input_2)
+    assert test_output_2 == "account Weeks "
```

Comment:
why is there a " " at the end after the split ? Is this prt of the defined functionality too ? 
Can we also add few more assert statements 
- one without any camelcase in the text
- one with multiple camelcase words in the text

Reply:
> why is there a " " at the end after the split ? Is this prt of the defined functionality too ?

Yeah it's part of it

> Can we also add few more assert statements

Got it